empyrical/stats.py

```diff
@@ -1005,6 +1005,46 @@ def cagr(returns, period=DAILY, annualization=None):
     return ending_value ** (1. / no_years) - 1
 
 
+def value_at_risk(returns, cutoff=0.05):
+    """
+    Parameters
+    ----------
+    returns : pandas.Series or 1-D numpy.array
+        Non-cumulative daily returns.
+    cutoff : float, optional
+        Decimal representing the percentage cutoff for the bottom percentile of
+        returns. Defaults to 0.05.
+
+    Returns
+    -------
+    VaR : float
+        The VaR value.
+    """
+    return np.percentile(returns, 100 * cutoff)
+
+
+def conditional_value_at_risk(returns, cutoff=0.05):
+    """
+    Parameters
+    ----------
+    returns : pandas.Series or 1-D numpy.array
+        Non-cumulative daily returns.
+    cutoff : float, optional
+        Decimal representing the percentage cutoff for the bottom percentile of
+        returns. Defaults to 0.05.
+
+    Returns
+    -------
+    CVaR : float
+        The CVaR value.
+    """
+    # PERF: Instead of using the 'value_at_risk' function to find the cutoff
+    # value, which requires a call to numpy.percentile, determine the cutoff
+    # index manually and partition out the lowest returns values.
+    num_low_days = int((len(returns) - 1) * cutoff) + 1
+    return np.mean(np.partition(returns, num_low_days - 1)[:num_low_days])
+
+
```

Comment:
The implementations of these functions look correct to me.  We should probably update the docstrings to explain what these functions are actually for and where/when you would want to use them.  For example, for cvar, we might want something like:

```python
def conditional_value_at_risk(returns, cutoff=0.05):
    """
    Conditional value at risk (CVaR) of a returns stream.

    CVaR measures the expected single-day returns of an asset on that asset's
    worst-performing day, where "worst-performing" is defined as falling below
    ``cutoff`` as a percentile of all daily returns.

    Parameters
    ----------
    ...
    """
```

cc @marketneutral @CaptainKanuk @jmccorriston for thoughts on language here.